### PR TITLE
EVG-2784 Use task groups for lint task

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -56,15 +56,14 @@ lintArgs += --vendored-linters --enable-gc
 #   gotype produces false positives because it reads .a files which
 #   are rarely up to date.
 lintArgs += --disable="gotype" --disable="gas" --disable="gocyclo" --disable="maligned"
-lintArgs += --disable="golint" --disable="goconst" --disable="dupl"
+lintArgs += --disable="golint" --disable="goconst" --disable="dupl" --disable="megacheck"
```
Comment (richardsamuels) [inline makefile]: Whoa, we're disabling megacheck globally here?

**Comment by bsamek** [comment]
This is blocked on a deploy to prod, since the config is invalid prior to dc1bae61502d1c2a3d92453fa2514427af79778d.

**Comment by tychoish** [comment]
how did this not run a patch build?

File: makefile (hunk 1 above)
Comment (bsamek) [reply makefile]: Yeah, I'm going to fold megacheck back in next in EVG-2803.

**Comment by bsamek** [comment]
I'm going to close this but not merge it, since tasks are not displayed for patch builds.

**Pull request closed without merging**